CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Migration notes:
 
 Enhancements:
 
+* Automatic module name added to Runtime Attach library to support Modular Java.
```

Comment:
can you create a new section at the top for 3.4.0-BETA.2, and move this there?

Reply:
done